sql/logictest: TestParallel failed

sql/logictest.TestParallel [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9632757?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9632757?buildTab=artifacts#/) on master @ [e7cac13f62d69373bc3a025d38de03ea2a02ee35](https://github.com/cockroachdb/cockroach/commits/e7cac13f62d69373bc3a025d38de03ea2a02ee35):


```

goroutine 460298 lock 0xc004c56650
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:703 kvserver.(*Replica).handleRaftReady ??? <<<<<
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:702 kvserver.(*Replica).handleRaftReady ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_raft.go:644 kvserver.(*Store).processReady ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:394 kvserver.(*raftSchedulerShard).worker ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:299 kvserver.(*raftScheduler).Start.func2 ???
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 stop.(*Stopper).RunAsyncTaskEx.func2 ???

goroutine 460569 lock 0xc0154efef8
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1184 kvserver.(*Replica).tick ??? <<<<<
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1183 kvserver.(*Replica).tick ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_raft.go:673 kvserver.(*Store).processTick ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:386 kvserver.(*raftSchedulerShard).worker ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:299 kvserver.(*raftScheduler).Start.func2 ???
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 stop.(*Stopper).RunAsyncTaskEx.func2 ???

goroutine 460615 lock 0xc0159de650
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1182 kvserver.(*Replica).tick ??? <<<<<
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1181 kvserver.(*Replica).tick ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_raft.go:673 kvserver.(*Store).processTick ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:386 kvserver.(*raftSchedulerShard).worker ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:299 kvserver.(*raftScheduler).Start.func2 ???
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 stop.(*Stopper).RunAsyncTaskEx.func2 ???

goroutine 517178 lock 0xc009f69e40
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica.go:1272 kvserver.(*Replica).getLastReplicaDescriptors ??? <<<<<
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica.go:1271 kvserver.(*Replica).getLastReplicaDescriptors ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1686 kvserver.(*Replica).sendRaftMessage ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1526 kvserver.(*Replica).sendRaftMessages ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:1506 kvserver.(*replicaSyncCallback).OnLogSync ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/logstore/logstore.go:226 logstore.(*LogStore).storeEntriesAndCommitBatch ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/logstore/logstore.go:138 logstore.(*LogStore).StoreEntries ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:958 kvserver.(*Replica).handleRaftReadyRaftMuLocked ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:704 kvserver.(*Replica).handleRaftReady ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_raft.go:644 kvserver.(*Store).processReady ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:394 kvserver.(*raftSchedulerShard).worker ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:299 kvserver.(*raftScheduler).Start.func2 ???
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 stop.(*Stopper).RunAsyncTaskEx.func2 ???

goroutine 33801 lock 0xc00702d9d0
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:703 kvserver.(*Replica).handleRaftReady ??? <<<<<
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/replica_raft.go:702 kvserver.(*Replica).handleRaftReady ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/store_raft.go:644 kvserver.(*Store).processReady ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:394 kvserver.(*raftSchedulerShard).worker ???
github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:299 kvserver.(*raftScheduler).Start.func2 ???
github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:470 stop.(*Stopper).RunAsyncTaskEx.func2 ???



```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallel.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-27062

sql/logictest.TestParallel [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9787433?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9787433?buildTab=artifacts#/) on master @ [a9ce55af2ff225264fcbdf4ffe0741a22858f605](https://github.com/cockroachdb/cockroach/commits/a9ce55af2ff225264fcbdf4ffe0741a22858f605):


```
W230425 15:41:15.362157 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470051  failed to flush statement statistics: flush statement 14315065430772624722's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.364699 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470052  failed to flush statement statistics: flush statement 16037756241246389521's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.366707 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470053  failed to flush statement statistics: flush statement 16037735350525453646's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.369167 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470054  failed to flush statement statistics: flush statement 6957563633013263361's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.371232 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470055  failed to flush statement statistics: flush statement 11336305007606786728's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.373548 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470056  failed to flush statement statistics: flush statement 18097629588326401083's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.375705 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470057  failed to flush statement statistics: flush statement 10899567374407408375's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.378341 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470058  failed to flush statement statistics: flush statement 2975591048621042534's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.380850 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470059  failed to flush statement statistics: flush statement 1464507226451452290's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.382961 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470060  failed to flush statement statistics: flush statement 18196689053367598556's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.385074 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470061  failed to flush statement statistics: flush statement 8931725617162097219's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.387401 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470062  failed to flush statement statistics: flush statement 8931739910813263976's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.389220 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470063  failed to flush statement statistics: flush statement 4553633081865308432's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.391125 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470064  failed to flush statement statistics: flush statement 9099898713836500970's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.392934 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470065  failed to flush statement statistics: flush statement 12215130198365885682's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.394748 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470066  failed to flush statement statistics: flush statement 17033636905401300538's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.397367 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470067  failed to flush statement statistics: flush statement 4407173395255379546's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.399647 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470068  failed to flush statement statistics: flush statement 8493476170941479432's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.401872 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470069  failed to flush statement statistics: flush statement 1319950413697517657's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.404017 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470070  failed to flush statement statistics: flush statement 6348203192952515288's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.407162 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470071  failed to flush statement statistics: flush statement 9098158856387001142's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.409738 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470072  failed to flush statement statistics: flush statement 5360193600302795791's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.411678 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470073  failed to flush statement statistics: flush statement 12740016567863122526's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.414101 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470074  failed to flush statement statistics: flush statement 5186423015962516865's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.416422 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470075  failed to flush statement statistics: flush statement 12017867313612103751's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.419003 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470076  failed to flush statement statistics: flush statement 18445459025402006891's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.421515 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470077  failed to flush statement statistics: flush statement 3321422013729747674's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.423371 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470078  failed to flush statement statistics: flush statement 4665035980191481368's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.425410 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470079  failed to flush statement statistics: flush statement 4648104696254794225's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.427918 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470080  failed to flush statement statistics: flush statement 18438780628890563601's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.429941 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470081  failed to flush statement statistics: flush statement 13311428551855483580's statistics: insert-stmt-stats: node unavailable; try another peer
W230425 15:41:15.432721 214158540 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 470082  failed to flush statement statistics: flush statement 16956964122461726811's statistics: insert-stmt-stats: node unavailable; try another peer
I230425 15:41:18.939020 214218625 util/stop/stopper.go:560  [n2] 470083  quiescing...
I230425 15:41:18.939080 214218675 util/stop/stopper.go:560  [n3] 470084  quiescing...
W230425 15:41:20.905466 587276 ts/db.go:208  [T1,n3,ts-poll] 470085  error writing time series data: operation "ts-poll" timed out after 1m0.005s (given timeout 1m0s): split failed while applying backpressure to Merge [/System/tsd/cr.node.admission.elastic_cpu.max_available_nanos/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-root-start.normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv.locking-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv.high-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv.normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.elastic-cpu/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.errored.elastic-cpu.normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.elastic_cpu.utilization_limit/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-sql-response/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-leaf-start.locking-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-kv-response/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.elastic_cpu.utilization/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.elastic_cpu.acquired_nanos/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.elastic_cpu.returned_nanos/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv-stores.normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv-stores.bulk-normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.kv-stores.locking-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.errored.kv/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-root-start.locking-pri/10s/2023-04-25T15:00:00Z/3,/Min),... 9 skipped ..., Merge [/System/tsd/cr.node.admission.admitted.sql-kv-response.locking-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-leaf-start/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-sql-response.normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.admitted.sql-leaf-start.normal-pri/10s/2023-04-25T15:00:00Z/3,/Min), Merge [/System/tsd/cr.node.admission.errored.elastic-cpu.bulk-normal-pri/10s/2023-04-25T15:00:00Z/3,/Min) on range r4:/System/tsd{-/cr.node.admission.errored.kv-stores/10s/2023-04-25T14:00:00Z/2} [(n1,s1):1, (n3,s3):2, (n2,s2):3, next=4, gen=6]: split at key /System/tsd/cr.node.admission.admitted.kv.normal-pri/10s/2023-04-25T14:00:00Z/1 failed: fetching current range descriptor value: failed to send RPC: sending to all replicas failed; last error: [NotLeaseHolderError] lease acquisition canceled because node is stopping; r4: replica (n3,s3):2 not lease holder; lease holder unknown
W230425 15:41:20.905466 587276 ts/db.go:208  [T1,n3,ts-poll] 470085 +HINT: For help understanding this error and troubleshooting, visit:
W230425 15:41:20.905466 587276 ts/db.go:208  [T1,n3,ts-poll] 470085 +
W230425 15:41:20.905466 587276 ts/db.go:208  [T1,n3,ts-poll] 470085 +    https://www.cockroachlabs.com/docs/stable/common-errors.html#split-failed-while-applying-backpressure-are-rows-updated-in-a-tight-loop
W230425 15:41:20.906401 587276 ts/db.go:211  [T1,n3,ts-poll] 470086  node unavailable; try another peer
W230425 15:42:10.556693 531320 ts/db.go:208  [T1,n2,ts-poll] 470087  error writing time series data: operation "ts-poll" timed out after 1m0.003s (given timeout 1m0s): failed to connect to n1 at 127.0.0.1:33135: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
I230425 15:42:10.560928 12678 server/server_controller_orchestration.go:263  [T1,n1] 470088  server controller shutting down ungracefully
I230425 15:42:10.561023 12678 server/server_controller_orchestration.go:274  [T1,n1] 470089  waiting for tenant servers to report stopped
W230425 15:42:10.569080 12678 server/server_sql.go:1693  [T1,n1] 470090  server shutdown without a prior graceful drain
I230425 15:42:10.575985 12678 server/server_controller_orchestration.go:263  [T1,n2] 470091  server controller shutting down ungracefully
I230425 15:42:10.576081 12678 server/server_controller_orchestration.go:274  [T1,n2] 470092  waiting for tenant servers to report stopped
W230425 15:42:10.581925 12678 server/server_sql.go:1693  [T1,n2] 470093  server shutdown without a prior graceful drain
I230425 15:42:10.595032 12678 server/server_controller_orchestration.go:263  [T1,n3] 470094  server controller shutting down ungracefully
I230425 15:42:10.595144 12678 server/server_controller_orchestration.go:274  [T1,n3] 470095  waiting for tenant servers to report stopped
W230425 15:42:10.602039 12678 server/server_sql.go:1693  [T1,n3] 470096  server shutdown without a prior graceful drain
    --- FAIL: TestParallel/create_stats (4958.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallel.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/logictest.TestParallel [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10049795?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/10049795?buildTab=artifacts#/) on master @ [2e2e5c138e43c79d1d723e72326da05b6c5a1d84](https://github.com/cockroachdb/cockroach/commits/2e2e5c138e43c79d1d723e72326da05b6c5a1d84):


```
W230512 13:37:20.292542 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19238  failed to flush statement statistics: flush statement 18196689053367598556's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.324630 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19239  failed to flush statement statistics: flush statement 10557014157430548028's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.354863 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19240  failed to flush statement statistics: flush statement 4140655024818416420's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.385116 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19241  failed to flush statement statistics: flush statement 6348203192952515288's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.416597 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19242  failed to flush statement statistics: flush statement 1464507226451452290's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.446496 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19243  failed to flush statement statistics: flush statement 10016303749509938257's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.477361 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19244  failed to flush statement statistics: flush statement 5079278749342244674's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.507245 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19245  failed to flush statement statistics: flush statement 1143961512702373112's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.536849 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19246  failed to flush statement statistics: flush statement 14621555150568216041's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.562987 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19247  failed to flush statement statistics: flush statement 14621578240312408326's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.588295 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19248  failed to flush statement statistics: flush statement 2417597999126952429's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.622045 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19249  failed to flush statement statistics: flush statement 5769566532107836724's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.648275 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19250  failed to flush statement statistics: flush statement 3871933716889782421's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.676968 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19251  failed to flush statement statistics: flush statement 16869474147472528604's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.703906 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19252  failed to flush statement statistics: flush statement 7226321729662032714's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.735308 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19253  failed to flush statement statistics: flush statement 18387747675008746556's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.762005 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19254  failed to flush statement statistics: flush statement 7726690590260747417's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.792523 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19255  failed to flush statement statistics: flush statement 7477005344783216079's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.817399 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19256  failed to flush statement statistics: flush statement 14921957443287757346's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.848282 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19257  failed to flush statement statistics: flush statement 16956964122461726811's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.875465 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19258  failed to flush statement statistics: flush statement 1879849972839045049's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.903366 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19259  failed to flush statement statistics: flush statement 14490211381980301883's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.933474 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19260  failed to flush statement statistics: flush statement 16869474147472528604's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.961674 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19261  failed to flush statement statistics: flush statement 16037756241246389521's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:20.989137 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19262  failed to flush statement statistics: flush statement 14315065430772624722's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.021341 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19263  failed to flush statement statistics: flush statement 16043164175446835940's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.049286 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19264  failed to flush statement statistics: flush statement 7227629162916973317's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.077904 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19265  failed to flush statement statistics: flush statement 11745958131368346088's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.103157 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19266  failed to flush statement statistics: flush statement 11745958131368346088's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.129745 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19267  failed to flush statement statistics: flush statement 3321422013729747674's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.157184 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19268  failed to flush statement statistics: flush statement 4665035980191481368's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.187890 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19269  failed to flush statement statistics: flush statement 12215130198365885682's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.219899 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19270  failed to flush statement statistics: flush statement 9099898713836500970's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.251107 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19271  failed to flush statement statistics: flush statement 18438780628890563601's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.277287 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19272  failed to flush statement statistics: flush statement 8594249169360073544's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.307358 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19273  failed to flush statement statistics: flush statement 3193856872155801960's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.338061 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19274  failed to flush statement statistics: flush statement 3419408685265610513's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.364841 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19275  failed to flush statement statistics: flush statement 12354992361753969940's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.394522 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19276  failed to flush statement statistics: flush statement 12354992361753969944's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
W230512 13:37:21.426305 55619566 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 19277  failed to flush statement statistics: flush statement 10491715399859778029's statistics: insert-stmt-stats: failed to connect to n3 at 127.0.0.1:37213: grpc: refusing to dial; node is quiescing [code 7/PermissionDenied]
I230512 13:37:21.442378 12584 server/server_controller_orchestration.go:263  [T1,n1] 19278  server controller shutting down ungracefully
I230512 13:37:21.442442 12584 server/server_controller_orchestration.go:274  [T1,n1] 19279  waiting for tenant servers to report stopped
W230512 13:37:21.456354 12584 server/server_sql.go:1694  [T1,n1] 19280  server shutdown without a prior graceful drain
I230512 13:37:21.459115 12584 server/server_controller_orchestration.go:263  [T1,n2] 19281  server controller shutting down ungracefully
I230512 13:37:21.459199 12584 server/server_controller_orchestration.go:274  [T1,n2] 19282  waiting for tenant servers to report stopped
W230512 13:37:21.468385 12584 server/server_sql.go:1694  [T1,n2] 19283  server shutdown without a prior graceful drain
I230512 13:37:21.471064 12584 server/server_controller_orchestration.go:263  [T1,n3] 19284  server controller shutting down ungracefully
I230512 13:37:21.471134 12584 server/server_controller_orchestration.go:274  [T1,n3] 19285  waiting for tenant servers to report stopped
W230512 13:37:21.482331 12584 server/server_sql.go:1694  [T1,n3] 19286  server shutdown without a prior graceful drain
    --- FAIL: TestParallel/create_stats (1195.53s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallel.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/logictest/parallel_test.go
+++ b/pkg/sql/logictest/parallel_test.go
@@ -195,6 +195,8 @@ func (t *parallelTest) setup(ctx context.Context, spec *parTestSpec) {
 		sql.DistSQLClusterExecMode.Override(ctx, &st.SV, int64(mode))
 		// Disable automatic stats - they can interfere with the test shutdown.
 		stats.AutomaticStatisticsClusterMode.Override(ctx, &st.SV, false)
+		stats.UseStatisticsOnSystemTables.Override(ctx, &st.SV, false)
+		stats.AutomaticStatisticsOnSystemTables.Override(ctx, &st.SV, false)
 	}
 
 	t.clients = make([][]*gosql.DB, spec.ClusterSize)

--- a/pkg/sql/logictest/testdata/parallel_test/create_stats/test.yaml
+++ b/pkg/sql/logictest/testdata/parallel_test/create_stats/test.yaml
@@ -3,7 +3,7 @@
 
 cluster_size: 3
 
-range_split_size: 32768
+range_split_size: 67108864
 
 run:
    # First run setup

